runtime/Cargo.toml

```diff
@@ -51,8 +51,8 @@ num-bigint = "0.4"
 num-traits = "0.2.14"
 bech32 = "0.8.1"
```

Comment:
Tendermint bumps seem to be missing

Reply:
Yeah because 0.23.1 implies 0.23.5 by default. Bad Tendermint, putting breaking changes into a patch release. Will update Cargo.toml as well.

Reply:
Done.